sdk/core/azure-core/spotbugs-exclude.xml

```diff
@@ -0,0 +1,376 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Long term we should decentralize all spotbugs and checkstyle suppressions and let individual libraries maintain it. Current global suppression is getting too big and unmanageable. Also, suppressions are never cleaned up and hard to track which libraries are still violating these rules.